userdbctl: add page (#10662)
tldr-pages/tldr: pages/linux/userdbctl.md

# userdbctl

> > Inspect user, groups and group memberships of the system.

Reviewer: ```suggestion
> Inspect users, groups and group memberships on the system.
```

> > To inspect user, groups and group memberships of the system.

Reviewer 2: How about this?

```suggestion
> Inspect user, groups and group memberships of the system.
```

> More information: <https://www.freedesktop.org/software/systemd/man/userdbctl.html>.

> - List all known users records:

Reviewer 2: ```suggestion
- List all known user records:
```

`userdbctl user`

- Show details of a specific user:

> `userdbctl user "{{username}}"`

Reviewer: ```suggestion
`userdbctl user {{username}}`
```

- List all known groups:

`userdbctl group`

- Show details of a specific group:

> `userdbctl group "{{groupname}}"`

Reviewer: ```suggestion
`userdbctl group {{groupname}}`
```
